libr/core/cmd_meta.c

```diff
@@ -953,6 +954,7 @@ static int cmd_meta(void *data, const char *input) {
 	case 'd': /* "Cd" data */
 	case 'm': /* Cm magic */
 	case 'f': /* Cf formatted */
+	case 't': /* Ct type analysis commnets */
 		cmd_meta_hsdmf (core, input);
```

Comment:
might as well rename the function

Reply:
yes but please rename it to something which is not based on the commands otherwise next time we add a new C type, you need to rename the function again. Something like `cmd_meta_types`, `cmd_meta_others` or something else.